e2e/test_operator_upgrades/operator_upgrades_test.go

```diff
@@ -386,6 +405,72 @@ var _ = Describe("Operator Upgrades", Label("e2e"), func() {
 		fixtures.GenerateUpgradeTableEntries(testOptions),
 	)
 
+	DescribeTable(
+		"upgrading a cluster where a storage and multiple stateless processes get restarted during the staging phase",
```

Comment:
This seems to be a completely unrelated test case to the PR name? Can we open a new PR for this test?

Reply:
As I mentioned above, extended the scope of this PR to cover all these cases.